spec/tasks/translation_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'spec_helper'
+require_relative '../../spec/support/tasks'  
+
+describe 'rake tasks' do
+  before do
+    fake_rails = double()
+    fake_rails.stub(:root) { Pathname.new('spec') }
+    stub_const("Rails", fake_rails)
+  end
+
+  describe 'translation:add_missing', type: :task do
+    let(:task) { Rake::Task["translation:add_missing"] }
+  
+    it 'is executed' do
+      expect { task.execute }.to output.to_stdout
+    end
+
+    it 'triggers i18n task' do
+      allow(I18n::Tasks::CLI).to receive(:start)
+      task.execute
+      expect(I18n::Tasks::CLI).to have_received(:start)
```

Comment:
We should also check that it's receiving the right argument, e.g.

```suggestion
      expect(I18n::Tasks::CLI).to have_received(:start).with(["add-missing"])
```

Reply:
@ChrisBAshton Have added those, thank you 👍

Reply:
Thanks! You can remove line 21 now, we only want the more specific line 22.